node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
delete

Reply:
yeah was wondering how this slipped past gitignore, maybe its in the root and not `website`